<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bus CAN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>881</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,3,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QWidget" name="SensorValues" native="true">
       <widget class="QGroupBox" name="windSpeedvalue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>341</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>WindSpeed (km/h)</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QLabel" name="windSpeedLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="TemperatureValue">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>0</y>
          <width>341</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Temperature (°C)</string>
        </property>
        <widget class="QLabel" name="temperaturelabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="HumidityValue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>341</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Humidity (%)</string>
        </property>
        <widget class="QLabel" name="HumidityLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="PressureValue">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>60</y>
          <width>341</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Pressure (hPa)</string>
        </property>
        <widget class="QLabel" name="PressureLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QWidget" name="threeDbox" native="true">
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>300</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="MotorBox" native="true">
       <widget class="QLineEdit" name="MotorSpeed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>261</width>
          <height>28</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Motor Speed</string>
        </property>
       </widget>
       <widget class="QPushButton" name="setSpeed">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="OnOffButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(26, 141, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QPushButton" name="autoToggleButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 110, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="DistanceBrightnessBox" native="true">
       <widget class="QRadioButton" name="DistanceRadio">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Distance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="BrightnessRadio">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>70</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="DistanceBrightnessValue">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>301</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="uniteLabel">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>24</y>
          <width>57</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QWidget" name="widget" native="true">
       <widget class="QGroupBox" name="PsiValue">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>40</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Psi (Ψ)</string>
        </property>
        <widget class="QLabel" name="PsiLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="ThetaValue">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>40</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Theta (θ)</string>
        </property>
        <widget class="QLabel" name="ThetaLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="phiValue">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Phi</string>
        </property>
        <widget class="QLabel" name="phiLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>